### DISCUSSION TITLE
TSSplot error

### DISCUSSION THREAD

**Discussion by ChengyueT** [stuart-lab/signac#961] [Q&A]
when I run command, 

> TSSPlot(pbmc, group.by = 'high.tss') + ggtitle("TSS enrichment score of D-0")

then error happened, 

> Error in h(simpleError(msg, call)) : 
> error in evaluating the argument 'x' in selecting a method for function 'colMeans': invalid character indexing.

I run Signac as tutorial but I do not know where did it go wrong. Do you think you could help me figure out what's causing this?

Thank you!

**Comment by timoast** [comment]
What's the output of `table(pbmc$high.tss)`?

**Reply by ChengyueT** [reply]
I checked it and find that there are NA in my preliminary seuratobject. I worked it out with the QC step by using a subset.
Thank you very much!
